schema.sql

```diff
@@ -330,6 +330,15 @@ CREATE TABLE IF NOT EXISTS `tile_store` (
   FOREIGN KEY (`house_id`) REFERENCES `houses` (`id`) ON DELETE CASCADE
 ) ENGINE=InnoDB;
 
+CREATE TABLE IF NOT EXISTS `live_casts` (
```

Comment:
I think it would be cleaner to modify `players_online` table to add columns: `cast_password` and `spectator_count` rather than adding a whole new table. If you absolutely think adding a new table is a good idea, it really should be `ENGINE=MEMORY`.